lib/index.js

```diff
@@ -14,14 +15,23 @@ function inspect(basePath, fileName, options) {
 
   try {
     // lockfile. usually composer.lock
-    composerLockObj = loadJsonFile(basePath, fileName);
+    var lockfilePath = path.resolve(basePath, fileName);
+    if (!fs.existsSync(lockfilePath)) {
+      throw new Error('Missing lock file at ' + lockfilePath);
+    }
+    composerLockObj = JSON.parse(fs.readFileSync(lockfilePath).toString());
```

Comment:
why not `require(lockfilePath)`?

Reply:
Good question! I’d rather bypass the `require` cache and other logic, and signal that all we need here is the parsed JSON. Also, if the file is malformed, a JSON parsing exception will be cleaner than a `require` issue, I think.